java/kotlin/generate-sources-build.xml

```diff
@@ -0,0 +1,20 @@
+<project name="generate-sources">
+    <echo message="Running protoc ..."/>
+    <mkdir dir="${generated.sources.dir}"/>
+    <exec executable="${protoc}">
+        <arg value="--java_out=${generated.sources.dir}"/>
+        <arg value="--proto_path=${protobuf.source.dir}"/>
```

Comment:
We should likely expose a kotlin version of all of these proto definitions as they are normally part of the public API.

Reply:
Done!